_data/simple-icons.json

```
@@ -7093,6 +7093,27 @@
             "hex": "CC2927",
             "source": "https://de.wikipedia.org/wiki/Datei:Microsoft_SQL_Server_Logo.svg"
         },
+        {
+            "title": "Microsoft SQL Server Management Studio",
+            "hex": "FEB900",
+            "source": "https://aleson-itc.com/wp-content/uploads/2020/03/server-m-studio.svg",
+            "aliases": {
+                "aka": [
+                    "SSMS",
+                    "SQL Server Management Studio"
```

```suggestion
                    "SSMS"
```
The website search will already show this icon without the additional alias here.